sdk/communication/communication-phone-numbers/src/mappers.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { SipTrunk } from "./models";
+import { SipTrunk as RestSipTrunk } from "./generated/src/siprouting/models";
+
+/**
+ * @internal
+ * Mapping SIP trunks REST model to SDK model
+ */
+export const mapTrunks = (
```

Comment:
nit: I would call these transform rather than map. Map is a concept of its own in the generated code, so it may be easy to get confused

Reply:
Renamed.
mapTrunks -> transformFromRestModel
mapTrunksToRestModel -> transformIntoRestModel